vuestorefront/magento2
# fix: currency cookie breaks the application

## Description
currency cookie breaks the application

## Related Issue





## Motivation and Context


## How Has This Been Tested?
- Open up the app and switch to a store with a currency that is not available in your current store, eg. if you are on the default store with just USD, switch to the German store.
- make sure that the currency cookie is set to EUR
- open default store with url (not by any app action like switching the store)
- observe that there will be the critical error

## Screenshots (if appropriate):

## Types of changes


## Checklist:
